File: tests/test-minecraft-osp/src/main/java/pize/tests/voxelgame/server/CommandUtils.java
package pize.tests.voxelgame.server;

import java.util.StringJoiner;

public class CommandUtils{
    
    public static String joinArgs(String[] args, int beginIndex){
        final StringJoiner joiner = new StringJoiner(" ");
        for(int i = beginIndex; i < args.length; i++)
            joiner.add(args[i]);
        
        return joiner.toString();
    }
    
}

File: tests/test-minecraft-osp/src/main/kotlin/pize/tests/voxelgame/server/CommandUtils.kt
package pize.tests.voxelgame.server

import java.util.*

object CommandUtils {
    fun joinArgs(args: Array<String?>, beginIndex: Int): String {
        val joiner = StringJoiner(" ")
        for (i in beginIndex until args.size) joiner.add(args[i])
        return joiner.toString()
    }
}
